llvm/lib/CodeGen/AsmPrinter/CodeViewDebug.cpp

```diff
@@ -807,6 +809,32 @@ void CodeViewDebug::emitObjName() {
   endSymbolRecord(CompilerEnd);
 }
 
+void CodeViewDebug::emitHotPatchInformation() {
```

Comment:
As mentionned before, maybe better to rename to `emitSecureHotPatchInformation()`

Reply:
> Would that tooling remain proprietary, or is it something that will be open-sourced as well, for example for NT drivers developpers?

In the near term, there are no plans on releasing these tools.  The only scenarios for these tools require signing using the same certs that we use for shipping OS components, so even if we provided the tools, there isn't currently a scenario where they could be used.

I could imagine that changing in the future, such as for device drivers, but I'm not in a position to speak for that direction.